pkg/smokescreen/testdata/acl.yaml

```diff
@@ -5,7 +5,7 @@
         project: security
         action: enforce
         allowed_domains:
-          - notarealhost.com
+          - notarealhost.notarealtld
```

Comment:
While it's unlikely that `.notarealtld` will ever become a real gTLD, `.test` exists explicitly for testing and will never have any hostnames/domains in it.